### PR TITLE
Style terms conditions

### PR DESCRIPTION
**Pull request by thdvbr**
- added styles to terms and conditions screen
- added styles to onboarding ( not 100% due to swiper library, TODO: make image resize available ) 
- added some translation to t&c and onboarding

### CODE REVIEW

File: src/screens/TermsAndConditions/index.js [hunk 1]
```diff
@@ -7,120 +7,139 @@ import {
...
-          {'\u2022'} Your use of this website and any dispute arising out of
-          such use of the website is subject to the laws of England, Northern
-          Ireland, Scotland and Wales.
-        </Text>
-        <Text style={styles.tcP}>
-          The use of this website is subject to the following terms of use
-        </Text>
+        <View style={styles.textContainer}>
+          <Text style={styles.tcP}>
+            Placeholder terms and conditions. If you continue to browse and use
```
[the remark this answers is not in the document]
Comment (thdvbr) [reply src/screens/TermsAndConditions/index.js:44]: moved text to another component

File: src/screens/TermsAndConditions/index.js [hunk 2]
```diff
@@ -7,120 +7,139 @@ import {
...
-    </View>
+        style={!isRead ? styles.buttonDisabled : styles.button}>
+        <StyledButtonText>{t('TermsAndConditions.button')}</StyledButtonText>
+      </StyledButton>
+    </StyledView>
   );
 }
 
-const {width, height} = Dimensions.get('window');
+const { width, height } = Dimensions.get('window');
```
[the remark this answers is not in the document]
Comment (thdvbr) [reply src/screens/TermsAndConditions/index.js:126]: made it global now

File: src/screens/Onboarding/index.js [hunk 3]
```diff
@@ -1,44 +1,60 @@
...
-export default function OnboardingScreen ({navigation}) {
-  const {t, i18n} = useTranslation();
+export default function OnboardingScreen({ navigation }) {
+  const { t, i18n } = useTranslation();
   return (
     <Onboarding
       onSkip={() => navigation.navigate('Login')}
       onDone={() => navigation.navigate('Login')}
+      titleStyles={styles.title}
+      imageContainerStyles={styles.image}
```
[the remark this answers is not in the document]
Comment (thdvbr) [reply src/screens/Onboarding/index.js:18]: nope

**Review by irina-baeva — APPROVED** [review]

**Comment by thdvbr** [comment]
- added alias paths: you can now access to the folders in the src by just using its name. eg) before: ../../styles/shared -> now: styles/shared/ ...

**Pull request merged**